paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -1186,26 +1167,6 @@ internal class CustomerSheetViewModel(
         }
     }
 
-    private fun buildFormObserver(
-        formArguments: FormArguments,
-        formViewModelSubcomponentBuilderProvider: Provider<FormViewModelSubcomponent.Builder>,
-        onFormDataUpdated: (data: FormViewModel.ViewData) -> Unit
-    ): () -> Unit {
-        val formViewModel = formViewModelSubcomponentBuilderProvider.get()
-            .formArguments(formArguments)
-            .showCheckboxFlow(flowOf(false))
-            .build()
-            .viewModel
-
-        return {
-            viewModelScope.launch {
-                formViewModel.viewDataFlow.collect { data ->
-                    onFormDataUpdated(data)
```

Comment:
I think this was a hack to wire up the data from the formViewModel.

But we already update this (far more reasonably) in `onFormFieldValuesCompleted`.